src/models/piranha_output.rs

```diff
@@ -14,15 +14,15 @@ Copyright (c) 2022 Uber Technologies, Inc.
 use std::path::PathBuf;
 
 use itertools::Itertools;
-use serde_derive::Serialize;
+use serde_derive::{Deserialize, Serialize};
 
 use crate::utilities::gen_py_str_methods;
 
 use super::{edit::Edit, matches::Match, source_code_unit::SourceCodeUnit};
 use pyo3::{prelude::pyclass, pymethods};
 
 /// A class to represent Piranha's output
-#[derive(Serialize, Debug, Clone, Default)]
+#[derive(Serialize, Debug, Clone, Default, Deserialize)]
```

Comment:
Had to add `Deserialize` for `PiranhaOutPutSummary` since in the new test case for `cli`. In this test we parse the output json (which is a `Vec<PiranhaOutputSummary>`) and compare with expected. Parsing the output json requires us to derive `Deserialize` for `PiranhaOutputSummary`